src/main/java/mate/academy/springboot/swagger/dto/ProductRequestDto.java

```diff
@@ -11,6 +11,6 @@ public class ProductRequestDto {
     @Size(min = 1, max = 120)
     private String title;
     @NotBlank
-    @Size(min = 0, max = 100000)
+    @Size(min = 0)
```

Comment:
Could we use size to the BigDecimal type? [See here ](https://docs.jboss.org/hibernate/beanvalidation/spec/2.0/api/javax/validation/constraints/Size.html)